doc/cookbook/source/examples/distance/canberra.rst

```diff
@@ -0,0 +1,39 @@
+==================
+Canberra Distance
+==================
+
+The Canberra distance between vectors P and Q is the similarity as measured by their angle.
```

Comment:
by their angle?
it's a weighted manhattan distance...

Reply:
Corrected.